docker/check_fast_version.py

```diff
@@ -0,0 +1,10 @@
+import sys
+
+try:
+    from gensim.models.word2vec_inner import FAST_VERSION
+
```

Comment:
Please add additional info for this script (like python version, numpy/scipy/gensim version) and create alias in docker for this script